conf/application.conf

```diff
@@ -89,7 +89,7 @@ auditing {
 }
 
 mongodb {
-  uri = "mongodb://localhost:27017/cip-phone-number-verification"
+  uri = "mongodb://localhost:27017/cip-phone-number-verification?serverselectiontimeoutms=10000"
```

Comment:
I think we should make the timeout configurable, i.e. as a separate config variable and not part of the url.  Then refer to it here in the url.

Reply:
Agreed, and well spotted, I forgot to raise PR for the app-config-{env} repos. Will raise some more shortly

Reply:
thanks for making them configurable now.  
One question, in other environments does mongodb always run on port 27017?  I think there are a range of ports you can choose from, 27018 is another one for example.  Therefore do you think you should make the port number configurable as well?

Reply:
ok fair enough.   No worries, just leave as is, the way you had it originally.